### PR TITLE
Migration: add `curl` code samples

### PR DESCRIPTION
**Pull request by guimachiavelli**

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -0,0 +1,1059 @@
...
+      "maxValuesPerFacet": 2
+    }'
+getting_started_pagination: |-
+  curl \
+    -X PATCH 'http://localhost:7700/indexes/movies/settings/pagination' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "maxTotalHits": 500
+    }'
+search_parameter_guide_matching_strategy_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:894]:
```suggestion
search_parameter_guide_matching_strategy_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X POST 'http://localhost:7700/indexes/movies/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "big fat liar",
+      "matchingStrategy": "last"
+    }'      
+search_parameter_guide_matching_strategy_2: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:902]:
```suggestion
search_parameter_guide_matching_strategy_2: |-
```

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X POST 'http://localhost:7700/indexes/movies/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "big fat liar",
+      "matchingStrategy": "all"
+    }'    
+date_guide_index_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:910]:
```suggestion
date_guide_index_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -x POST 'http://localhost:7700/indexes/games/documents' \
+    -h 'content-type: application/json' \
+    --data-binary @games.json
+date_guide_filterable_attributes_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:915]:
```suggestion
date_guide_filterable_attributes_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X PUT 'http://localhost:7700/indexes/games/settings/filterable-attributes' \
+    -H 'Content-Type: application/json' \
+    --data-binary '[
+      "release_timestamp"
+    ]'
+date_guide_filter_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:922]:
```suggestion
date_guide_filter_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 6]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X POST 'http://localhost:7700/indexes/games/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "",
+      "filter": "release_timestamp >= 1514761200 AND release_timestamp < 1672527600"
+    }'
+date_guide_sortable_attributes_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:930]:
```suggestion
date_guide_sortable_attributes_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 7]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X PUT 'http://localhost:7700/indexes/games/settings/sortable-attributes' \
+    -H 'Content-Type: application/json' \
+    --data-binary '[
+      "release_timestamp"
+    ]'
+date_guide_sort_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:937]:
```suggestion
date_guide_sort_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 8]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X POST 'http://localhost:7700/indexes/games/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "",
+      "sort": ["release_timestamp:desc"]
+    }'
+async_guide_filter_by_date_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:945]:
```suggestion
async_guide_filter_by_date_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 9]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X GET 'http://localhost:7700/tasks?afterEnqueuedAt=2020-10-11T11:49:53.000Z'
+async_guide_multiple_filters_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:948]:
```suggestion
async_guide_multiple_filters_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 10]
```diff
@@ -0,0 +1,1059 @@
...
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies&types=documentAdditionOrUpdate,documentDeletion&statuses=processing'
+async_guide_filter_by_ids_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:951]:
```suggestion
async_guide_filter_by_ids_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 11]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_canceled_by_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?canceledBy=9,15'      
+delete_tasks_1: |
+  curl \
+    -X DELETE 'http://localhost:7700/tasks?uids=1,2'
+cancel_tasks_1: |  
+  curl \
+    -X POST 'http://localhost:7700/tasks/cancel?uids=1,2'
+swap_indexes_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:972]:
```suggestion
swap_indexes_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 12]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_filter_by_index_uids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies'
+async_guide_canceled_by_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?canceledBy=9,15'      
+delete_tasks_1: |
+  curl \
+    -X DELETE 'http://localhost:7700/tasks?uids=1,2'
+cancel_tasks_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:969]:
```suggestion
cancel_tasks_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 13]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_filter_by_types_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?types=dumpCreation,indexSwap'
+async_guide_filter_by_index_uids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies'
+async_guide_canceled_by_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?canceledBy=9,15'      
+delete_tasks_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:966]:
```suggestion
delete_tasks_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 14]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_filter_by_statuses_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?statuses=failed,canceled'
+async_guide_filter_by_types_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?types=dumpCreation,indexSwap'
+async_guide_filter_by_index_uids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies'
+async_guide_canceled_by_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:963]:
```suggestion
async_guide_canceled_by_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 15]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_filter_by_ids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?uids=5,10,13'
+async_guide_filter_by_statuses_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?statuses=failed,canceled'
+async_guide_filter_by_types_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?types=dumpCreation,indexSwap'
+async_guide_filter_by_index_uids_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:960]:
```suggestion
async_guide_filter_by_index_uids_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 16]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_multiple_filters_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies&types=documentAdditionOrUpdate,documentDeletion&statuses=processing'
+async_guide_filter_by_ids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?uids=5,10,13'
+async_guide_filter_by_statuses_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?statuses=failed,canceled'
+async_guide_filter_by_types_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:957]:
```suggestion
async_guide_filter_by_types_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 17]
```diff
@@ -0,0 +1,1059 @@
...
+async_guide_filter_by_date_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?afterEnqueuedAt=2020-10-11T11:49:53.000Z'
+async_guide_multiple_filters_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?indexUids=movies&types=documentAdditionOrUpdate,documentDeletion&statuses=processing'
+async_guide_filter_by_ids_1: |
+  curl \
+    -X GET 'http://localhost:7700/tasks?uids=5,10,13'
+async_guide_filter_by_statuses_1: |
```
Comment (maryamsulemani97) [inline .code-samples.meilisearch.yaml:954]:
```suggestion
async_guide_filter_by_statuses_1: |-
```

**Review by maryamsulemani97 — COMMENTED** [review]
I updated some of the code samples from `code_sample_name: |` to `code_sample_name: |-`

**Review by dichotommy — APPROVED** [review]
🏎️

**Pull request merged**